abci/types/types.proto

```diff
@@ -83,8 +83,8 @@ message RequestBeginBlock {
 }
 
 enum CheckTxType {
-  New     = 0;
-  Recheck = 1;
+  NEW     = 0;
+  RECHECK = 1;
```

Comment:
Why these changes/is this explained in the changelog?

Reply:
this was to adhere to the linter where enum names are `upper_snake_case`. will specify in change log

Reply:
Ssssounds great, thanksss